# HG changeset patch
# User "Yann E. MORIN" <yann.morin.1998@anciens.enib.fr>
# Date 1322609122 -3600
# Node ID b6faa28e76e067be1306019b64120492f2fc655d
# Parent  85477445af5677617618564f1765c9f7722a1449
arch: rename the SUPPORT_XXX options

First, 'SUPPORT' should be spelled 'SUPPORTS'.

Second, 'SUPPORT_XXX' really means 'supports --with-xxx', so rename the
affected options accordingly. Update the affected archs to match the new
namings.

Reported-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Signed-off-by: "Yann E. MORIN" <yann.morin.1998@anciens.enib.fr>

diff --git a/config/arch/alpha.in b/config/arch/alpha.in
--- a/config/arch/alpha.in
+++ b/config/arch/alpha.in
@@ -3,7 +3,7 @@
 ## select ARCH_SUPPORTS_32
 ## select ARCH_DEFAULT_32
 ## select ARCH_USE_MMU
-## select ARCH_SUPPORT_CPU
-## select ARCH_SUPPORT_TUNE
+## select ARCH_SUPPORTS_WITH_CPU
+## select ARCH_SUPPORTS_WITH_TUNE
 ##
 ## help The Alpha architecture.
diff --git a/config/arch/arm.in b/config/arch/arm.in
--- a/config/arch/arm.in
+++ b/config/arch/arm.in
@@ -6,11 +6,11 @@
 ## select ARCH_DEFAULT_HAS_MMU
 ## select ARCH_SUPPORTS_BOTH_ENDIAN
 ## select ARCH_DEFAULT_LE
-## select ARCH_SUPPORT_ARCH
-## select ARCH_SUPPORT_CPU
-## select ARCH_SUPPORT_TUNE
-## select ARCH_SUPPORT_FPU
-## select ARCH_SUPPORT_SOFTFP
+## select ARCH_SUPPORTS_WITH_ARCH
+## select ARCH_SUPPORTS_WITH_CPU
+## select ARCH_SUPPORTS_WITH_TUNE
+## select ARCH_SUPPORTS_WITH_FPU
+## select ARCH_SUPPORTS_SOFTFP
 ##
 ## help The ARM architecture, as defined by:
 ## help     http://www.arm.com/
diff --git a/config/arch/arm.in.2 b/config/arch/arm.in.2
--- a/config/arch/arm.in.2
+++ b/config/arch/arm.in.2
@@ -49,4 +49,4 @@
     bool
     default y
     depends on ! ARCH_ARM_EABI
-    select ARCH_SUPPORT_ABI
+    select ARCH_SUPPORTS_WITH_ABI
diff --git a/config/arch/avr32.in b/config/arch/avr32.in
--- a/config/arch/avr32.in
+++ b/config/arch/avr32.in
@@ -4,10 +4,10 @@
 ## select ARCH_DEFAULT_32
 ## select ARCH_USE_MMU
 ## select ARCH_DEFAULT_BE
-## select ARCH_SUPPORT_ARCH
-## select ARCH_SUPPORT_CPU
-## select ARCH_SUPPORT_TUNE
-## select ARCH_SUPPORT_FPU
+## select ARCH_SUPPORTS_WITH_ARCH
+## select ARCH_SUPPORTS_WITH_CPU
+## select ARCH_SUPPORTS_WITH_TUNE
+## select ARCH_SUPPORTS_WITH_FPU
 ##
 ## help The AVR32 architecture, as defined by:
 ## help     http://www.atmel.com/products/avr32
diff --git a/config/arch/blackfin.in b/config/arch/blackfin.in
--- a/config/arch/blackfin.in
+++ b/config/arch/blackfin.in
@@ -3,9 +3,9 @@
 ## select ARCH_SUPPORTS_32
 ## select ARCH_DEFAULT_32
 ## select ARCH_DEFAULT_LE
-## select ARCH_SUPPORT_ARCH
-## select ARCH_SUPPORT_CPU
-## select ARCH_SUPPORT_TUNE
-## select ARCH_SUPPORT_FPU
+## select ARCH_SUPPORTS_WITH_ARCH
+## select ARCH_SUPPORTS_WITH_CPU
+## select ARCH_SUPPORTS_WITH_TUNE
+## select ARCH_SUPPORTS_WITH_FPU
 ##
 ## help The Blackfin architecture
diff --git a/config/arch/m68k.in b/config/arch/m68k.in
--- a/config/arch/m68k.in
+++ b/config/arch/m68k.in
@@ -5,6 +5,6 @@
 ## select ARCH_SUPPORTS_32
 ## select ARCH_DEFAULT_32
 ## select ARCH_DEFAULT_BE
-## select ARCH_SUPPORT_CPU
+## select ARCH_SUPPORTS_WITH_CPU
 ##
 ## help The m68k architecture
diff --git a/config/arch/mips.in b/config/arch/mips.in
--- a/config/arch/mips.in
+++ b/config/arch/mips.in
@@ -6,8 +6,8 @@
 ## select ARCH_USE_MMU
 ## select ARCH_SUPPORTS_BOTH_ENDIAN
 ## select ARCH_DEFAULT_BE
-## select ARCH_SUPPORT_ARCH
-## select ARCH_SUPPORT_TUNE
+## select ARCH_SUPPORTS_WITH_ARCH
+## select ARCH_SUPPORTS_WITH_TUNE
 ##
 ## help The MIPS architecture, as defined by:
 ## help     http://www.mips.com/
diff --git a/config/arch/powerpc.in b/config/arch/powerpc.in
--- a/config/arch/powerpc.in
+++ b/config/arch/powerpc.in
@@ -4,9 +4,9 @@
 ## select ARCH_SUPPORTS_64 if EXPERIMENTAL
 ## select ARCH_DEFAULT_32
 ## select ARCH_USE_MMU
-## select ARCH_SUPPORT_ABI
-## select ARCH_SUPPORT_CPU
-## select ARCH_SUPPORT_TUNE
+## select ARCH_SUPPORTS_WITH_ABI
+## select ARCH_SUPPORTS_WITH_CPU
+## select ARCH_SUPPORTS_WITH_TUNE
 ##
 ## help The PowerPC architecture, as defined by:
 ## help     http://www.ibm.com/developerworks/eserver/articles/archguide.html
diff --git a/config/arch/s390.in b/config/arch/s390.in
--- a/config/arch/s390.in
+++ b/config/arch/s390.in
@@ -6,7 +6,7 @@
 ## select ARCH_SUPPORTS_64
 ## select ARCH_DEFAULT_32
 ## select ARCH_USE_MMU
-## select ARCH_SUPPORT_FPU
+## select ARCH_SUPPORTS_WITH_FPU
 ##
 ## help Defined by IBM in:
 ## help     32-bit ESA/390       : http://publibfp.boulder.ibm.com/cgi-bin/bookmgr/BOOKS/dz9ar008/CCONTENTS
diff --git a/config/arch/sparc.in b/config/arch/sparc.in
--- a/config/arch/sparc.in
+++ b/config/arch/sparc.in
@@ -4,8 +4,8 @@
 ## select ARCH_SUPPORTS_64
 ## select ARCH_DEFAULT_32
 ## select ARCH_USE_MMU
-## select ARCH_SUPPORT_CPU
-## select ARCH_SUPPORT_TUNE
+## select ARCH_SUPPORTS_WITH_CPU
+## select ARCH_SUPPORTS_WITH_TUNE
 ##
 ## help The SUN SPARC architecture, as defined by:
 ## help     32 bit: http://www.sparc.org/standards/V8.pdf
diff --git a/config/arch/x86.in b/config/arch/x86.in
--- a/config/arch/x86.in
+++ b/config/arch/x86.in
@@ -4,9 +4,9 @@
 ## select ARCH_SUPPORTS_64
 ## select ARCH_DEFAULT_32
 ## select ARCH_USE_MMU
-## select ARCH_SUPPORT_ARCH
-## select ARCH_SUPPORT_CPU
-## select ARCH_SUPPORT_TUNE
+## select ARCH_SUPPORTS_WITH_ARCH
+## select ARCH_SUPPORTS_WITH_CPU
+## select ARCH_SUPPORTS_WITH_TUNE
 ##
 ## help The x86 architecture, as defined by:
 ## help     32-bit (ia32) : http://www.intel.com/
diff --git a/config/target.in b/config/target.in
--- a/config/target.in
+++ b/config/target.in
@@ -10,12 +10,12 @@
 config ARCH_SUPPORTS_BOTH_ENDIAN
 config ARCH_SUPPORTS_32
 config ARCH_SUPPORTS_64
-config ARCH_SUPPORT_ARCH
-config ARCH_SUPPORT_ABI
-config ARCH_SUPPORT_CPU
-config ARCH_SUPPORT_TUNE
-config ARCH_SUPPORT_FPU
-config ARCH_SUPPORT_SOFTFP
+config ARCH_SUPPORTS_WITH_ARCH
+config ARCH_SUPPORTS_WITH_ABI
+config ARCH_SUPPORTS_WITH_CPU
+config ARCH_SUPPORTS_WITH_TUNE
+config ARCH_SUPPORTS_WITH_FPU
+config ARCH_SUPPORTS_SOFTFP
 
 config ARCH_DEFAULT_HAS_MMU
 config ARCH_DEFAULT_BE
@@ -136,28 +136,28 @@
 #--------------------------------------
 comment "Target optimisations"
 
-config ARCH_SUPPORT_ARCH
+config ARCH_SUPPORTS_WITH_ARCH
     bool
 
-config ARCH_SUPPORT_ABI
+config ARCH_SUPPORTS_WITH_ABI
     bool
 
-config ARCH_SUPPORT_CPU
+config ARCH_SUPPORTS_WITH_CPU
     bool
 
-config ARCH_SUPPORT_TUNE
+config ARCH_SUPPORTS_WITH_TUNE
     bool
 
-config ARCH_SUPPORT_FPU
+config ARCH_SUPPORTS_WITH_FPU
     bool
 
-config ARCH_SUPPORT_SOFTFP
+config ARCH_SUPPORTS_SOFTFP
     bool
 
 config ARCH_ARCH
     string
     prompt "Architecture level"
-    depends on ARCH_SUPPORT_ARCH
+    depends on ARCH_SUPPORTS_WITH_ARCH
     default ""
     help
       GCC uses this name to determine what kind of instructions it can emit
@@ -177,7 +177,7 @@
 config ARCH_ABI
     string
     prompt "Generate code for the specific ABI"
-    depends on ARCH_SUPPORT_ABI
+    depends on ARCH_SUPPORTS_WITH_ABI
     default ""
     help
       Generate code for the given ABI.
@@ -194,7 +194,7 @@
 config ARCH_CPU
     string
     prompt "Emit assembly for CPU"
-    depends on ARCH_SUPPORT_CPU
+    depends on ARCH_SUPPORTS_WITH_CPU
     default ""
     help
       This specifies the name of the target processor. GCC uses this name
@@ -213,7 +213,7 @@
 config ARCH_TUNE
     string
     prompt "Tune for CPU"
-    depends on ARCH_SUPPORT_TUNE
+    depends on ARCH_SUPPORTS_WITH_TUNE
     default ""
     help
       This option is very similar to the ARCH_CPU option (above), except
@@ -236,7 +236,7 @@
 config ARCH_FPU
     string
     prompt "Use specific FPU"
-    depends on ARCH_SUPPORT_FPU
+    depends on ARCH_SUPPORTS_WITH_FPU
     default ""
     help
       On some targets (eg. ARM), you can specify the kind of FPU to emit
@@ -284,7 +284,7 @@
 config ARCH_FLOAT_SOFTFP
     bool
     prompt "softfp"
-    depends on ARCH_SUPPORT_SOFTFP
+    depends on ARCH_SUPPORTS_SOFTFP
     help
       Emit hardware floating point opcodes but use the software
       floating point calling convention.
